Userland/Libraries/LibWeb/HTML/Worker.cpp

```diff
@@ -41,9 +41,10 @@ JS::ThrowCompletionOr<void> Worker::initialize(JS::Realm& realm)
 void Worker::visit_edges(Cell::Visitor& visitor)
 {
     Base::visit_edges(visitor);
-    visitor.visit(m_document.ptr());
-    visitor.visit(m_implicit_port.ptr());
-    visitor.visit(m_outside_port.ptr());
+    visitor.visit(m_document);
+    visitor.visit(m_inner_settings);
```

Comment:
Eep, this one also belongs to the worker VM.
We really need to annotate these in some smarter way.

Reply:
Hmm, any ideas? Maybe for now, just something simple like `using BorrowPtr = GCPtr`, or any other number of names (`SharedPtr`/`WeakPtr`/etc)?